src/DIRAC/MonitoringSystem/Client/Types/PilotMonitoring.py

```diff
@@ -0,0 +1,26 @@
+from DIRAC.MonitoringSystem.Client.Types.BaseType import BaseType
+
+
+class PilotMonitoring(BaseType):
+    def __init__(self):
+
+        super(PilotMonitoring, self).__init__()
```

Comment:
In python3 you can simply do 
```suggestion
        super().__init__()
```